_pages/Statsboard/index.js

```diff
@@ -20,9 +39,31 @@ const Statsboard = (props) => {
     if (username) getPlayersStats();
   }, username);
 
+  console.log(stats);
+
+  const statsArr = [];
+
+  for (const property in stats) {
+    if (lang[property])
+      statsArr.push(
+        <div>
+          <p key={property}>
+            {' '}
+            {lang[property]} {stats[property]}
+          </p>
+        </div>,
+      );
+  }
```

Comment:
You could `.map` over the `stats` array here to get the same output. 

You may already be aware of this, but the `.map`, `.filter` and `.reduce` array methods will become your best friends, you can achieve most of the array manipulation you'll need with a combination of those three methods. Here is a good resource on it https://medium.com/poka-techblog/simplify-your-javascript-use-map-reduce-and-filter-bd02c593cc2d and there will be countless examples of all 3 of these in verticals repo!